src/app/public/modules/colorpicker/colorpicker.component.spec.ts

```diff
@@ -662,6 +662,20 @@ describe('Colorpicker Component', () => {
       verifyMenuVisibility();
     }));
 
+    it('should accept close colorpicker via messageStream.', fakeAsync(() => {
+      fixture.detectChanges();
+      tick();
+      component.sendMessage(SkyColorpickerMessageType.Open);
+      tick();
+      fixture.detectChanges();
+      tick();
+      component.sendMessage(SkyColorpickerMessageType.Close);
+      tick();
+      fixture.detectChanges();
+      tick();
+      verifyMenuVisibility(false);
```

Comment:
We should verify that the menu is visible after the `Open` message is sent.
```
verifyMenuVisibility(true);
```

Reply:
Done